src/main/java/net/pms/newgui/SelectRenderers.java

```diff
@@ -52,45 +58,41 @@ public void build() {
 		JPanel checkPanel = new JPanel();
 		checkPanel.applyComponentOrientation(ComponentOrientation.getOrientation(new Locale(configuration.getLanguage())));
 		add(checkPanel, BorderLayout.LINE_START);
-		allRenderers = new DefaultMutableTreeNode(Messages.getString("GeneralTab.13"));
-		DefaultMutableTreeNode renderersGroup = null;
-		String lastGroup = null;
-		String groupName;
-		boolean firstLoop = true;
-		for (String renderer : RendererConfiguration.getAllRenderersNames()) {
-			if (lastGroup != null && renderer.startsWith(lastGroup)) {
-				if (renderer.indexOf(" ") > 0 ) {
-					renderersGroup.add(new DefaultMutableTreeNode(renderer.substring(renderer.indexOf(" "))));
-				} else {
-					renderersGroup.add(new DefaultMutableTreeNode(renderer));
-				}
+		allRenderers = new SearchableMutableTreeNode(Messages.getString("GeneralTab.13"));
 
-			} else {
-				if (!firstLoop) {
-					allRenderers.add(renderersGroup);
+		Pattern pattern = Pattern.compile("^\\s*([^\\s]*) ?([^\\s].*?)?\\s*$");
```

Comment:
@valib Aargh! The original, tested regex was ```^\s*([^\s]*)\s*([^\s].*?)?\s*$```. During "final touches" I changed this to ```^\s*([^\s]*) ([^\s].*?)?\s*$``` to force a single space to comply with the "grouping" rules. I assumed this to be safe, and did not retest. It obviously isn't as it requires a space to get a match at all. The correct regex is ```^\s*([^\s]*) ?([^\s].*?)?\s*$```.